api/src/main/java/org/apache/iceberg/view/ViewVersion.java

```diff
@@ -58,4 +60,9 @@ public interface ViewVersion {
    * @return the list of view representations
    */
   List<ViewRepresentation> representations();
+
+  enum Operation {
```

Comment:
This seemed better to be an enum since these will be well defined by the Iceberg View specification.

Reply:
Not relevant anymore: refer to https://github.com/apache/iceberg/pull/6861#discussion_r1130428658